src/components/OrderPanel/OrderPanel.js

```diff
@@ -194,7 +194,8 @@ const OrderPanel = props => {
   } = props;
 
   const publicData = listing?.attributes?.publicData || {};
-  const { listingType, unitType, transactionProcessAlias = '' } = publicData || {};
+  const { listingType, unitType, transactionProcessAlias = '', availabilityType } =
+    publicData || {};
```

Comment:
The **availabilityType** is not saved to publicData (stockType is not there and this is similar feature).

So, check the availabilityType from listingTypeConfig (a bit similar fashion as `allowOrdersOfMultipleItems`)

Reply:
So, this is a communication failure: we should have had a meeting at the beginning of this customer-side project. Sorry :(